40_wa_pre_post_deaths.ipynb

```diff
@@ -0,0 +1,536 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "import pandas as pd\n",
+    "import altair as alt\n",
+    "import matplotlib.pyplot as plt\n",
+    "import statsmodels.api as sm\n",
+    "import statsmodels.formula.api as smf"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "deaths = pd.read_csv(\"05_cleaned_data/deaths_wa.csv\")"
```

Comment:
Yes thank you for the great advice! I added comments at the beginning of each code chunk in the updated files. Hope it would be helpful to navigate through the file